### ISSUE TITLE
(Question) How to make bots micspam?

### ISSUE THREAD

**Issue by JokesOnYouIAlreadyHaveALife** [nullworks/cathook#1355]
I don't have telegram, but I was wondering how to make my bots micspam? I have the micspam option for cathook enabled but how do I actually make them play music?

**Comment by iraizo** [comment]
https://steamcommunity.com/sharedfiles/filedetails/?id=1810325346

**Comment by Auditormadness9** [comment]
You can't, it only works for your tf2 for now.

**Comment by iraizo** [comment]
> You can't, it only works for your tf2 for now.

Yes you can, i imagine the feature doing +voicerecord automatically so you just pass trough your music

**Comment by Auditormadness9** [comment]
> > You can't, it only works for your tf2 for now.
> 
> Yes you can, i imagine the feature doing +voicerecord automatically so you just pass trough your music

It does not work on bots. We have discussed this with BenCat07 for months now.

**Comment by iraizo** [comment]
> It does not work on bots. We have discussed this with BenCat07 for months now.

Thats weird

**Comment by iraizo** [comment]
> > Thats weird
> 
> Ikr. Also, what difficulty label would you give to this? I'm curious cuz even TNE said he has no idea how to fix it. thinking

TNE has way more expierence with its source, so if he can't fix it I couldn't either.

**Comment by TotallyNotElite** [comment]
> I'm curious cuz even TNE said he has no idea how to fix it. thinking

Yea, I have no idea how to fix it of the top of my head, I'd need to do actual testing, but I'm not really interested in doing that right now.
